src/collective/mustread/interfaces.py

```diff
@@ -59,22 +59,20 @@ class ITracker(Interface):
     - mark_read(obj, 'maryjane')
     - has_read(obj, 'johndoe')
     - who_read(obj)
-
-    This is the initial implementation. It is not required (and at this stage
-    not even possible) to schedule a ``must_read(obj)`` to track reads.
-
-    Scheduled for later extension:
+    - uids_read('johndoe')
 
     2. Specify which users should read an object, with a deadline; then
        track reads on that object, and query which users have failed to
        meet the requirment of reading the object in time:
 
-    - must_read(obj, ['johndoe', 'maryjane'], nextweek)
+    - schedule_must_read(obj, ['johndoe', 'maryjane'], nextweek)
     - mark_read(obj, 'johndoe')
-    - who_unread(obj)
+    - who_must_read(obj)
+    - what_to_read(context)
+    - what_to_read(userid)
```

Comment:
done in https://github.com/collective/collective.mustread/pull/1/commits/73f1a3dd808f4ea49ec70f8cce08319cd30e39a2